lib/ebpd_utils.h

```diff
@@ -0,0 +1,30 @@
+#ifndef EBPD_UTILS_H_
+#define EBPD_UTILS_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/* Public utilities which can be used from userspace */
+
+/*
+ * API to load ebpf object code from a file into kernel
+ */
+extern int ebpd_load_xdp_prog (const char *filepath, int ifindex, void **handle);
```

Comment:
If ebpd_utils.h is converted to C++, these functions should be named according to the Google style guide, also use std::string for filename and std::string_view for buf below.

Reply:
Plan to retain it as C API